### PR TITLE
CI setup

### PR DESCRIPTION
**Pull request by bazarnyi**

### CODE REVIEW

**Comment by atatarenko** [comment]
The first thing I found when I looked at commit list
screen shot 2018-03-05 at 1 31 21 pm

It's a cool thing from some perspective. At the same time, they aren't informative at all. `Squash` is a git feature that might be useful for you. Check [it](https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git) out.

**Review by atatarenko — COMMENTED** [review]
Your changes look okay to me. I'm going to approve it right after I'm sure that commit history is well prepared.

File: .postcssrc.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+postcss: {}
```
Comment (atatarenko) [inline .postcssrc.yml:1]: Do we really need this file?

**Review by atatarenko — APPROVED** [review]
LGTM 👍

File: .postcssrc.yml (hunk 1 above)
Comment (bazarnyi) [reply .postcssrc.yml:1]: Unfortunately yes :(
For some reason assets compiling does not work on Linux (both CI and Heroku). 
Locally on my Mac evening looks fine. Don’t know the reason yet. I will add a task to backlog. Maybe at some point I will find other solution

**Pull request merged**